docs/src/main/sphinx/client/jdbc.md

```diff
@@ -246,3 +246,10 @@ may not be specified using both methods.
   - Sets the time zone for the session using the [time zone
     passed](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/ZoneId.html#of(java.lang.String)).
     Defaults to the timezone of the JVM running the JDBC driver.
+* - `legacyPreparedStatements`
+  - When set to `false`, prepared statements are executed calling a single
+    `EXECUTE IMMEDIATE` query instead of the standard `PREPARE <statement>`
+    followed by `EXECUTE <statement>`. Note that calling
```

Comment:
This should explain to users why they would want to change this setting. That it's faster (one less round-trip) but requires Trino server version >= 431.

Reply:
So why is the default not `false` .. that would make more sense right?

Reply:
It was decided to keep the legacy behavior at least for now to avoid to break any possible deployment. We may decide to change the default in the future.

Anyway, I've updated the documentation following your comments.

Reply:
Oh well.. I am not sure agree with the decision but it is what it is.